docs/resources/samples/automate-tasks-on-all-excel-files-in-folder.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Run a script on all Excel files in a folder'
+description: 'Learn how to run a script on all the Excel files in a folder on OneDrive for Business.'
+ms.date: 03/31/2021
+localization_priority: Normal
+---
+
+# Run a script on all Excel files in a folder
+
+This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business (can also be used for SharePoint folder).
```

Comment:
Let's make this two sentences.
```suggestion
This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business. It could also be used in a SharePoint folder.
```